lang/fr-fr.js

```diff
@@ -49,7 +49,11 @@ export default {
 	"components.meter-mixin.progressIndicator": "Indicateur de progrès",
 	"components.more-less.less": "moins",
 	"components.more-less.more": "plus",
+	"components.overflow-group.moreActions": "Plus d’actions",
```

Comment:
Not a major issue but this is something we normally correct manually -- the apostrophe here isn't the standard `'` one, but a specialized one that sneaks in occasionally.